shell/src/main/java/tachyon/shell/command/FileInfoCommand.java

```diff
@@ -54,7 +54,8 @@ void runCommand(TachyonURI path) throws IOException {
       throw new IOException(path + " is a directory path so does not have file blocks.");
     }
 
-    System.out.println(path + " with file id " + fd.getFileId() + " has the following blocks: ");
+    System.out.println(fInfo);
```

Comment:
oh, I see what you meant. Yes, this change is useful because it outputs more information such as persistence state in the file info, and therefore serves as convenience shell command to check if async persist has finished.